metaflow/metaflow_config.py

```diff
@@ -45,6 +45,13 @@ def from_conf(name, default=None):
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 
+##
+# Per run configuration
+###
+KFP_SDK_NAMESPACE = from_conf('KFP_SDK_NAMESPACE')
+KFP_SDK_API_NAMESPACE = from_conf('KFP_SDK_API_NAMESPACE')
+KFP_SDK_USERID = from_conf('KFP_SDK_USERID')
```

Comment:
Can this be METAFLOW_USER?

Reply:
@talebzeghmi I thought we already decided we will keep it `KFP_SDK_USERID` and deal with mapping KFP to MF names later.

Reply:
That is for namespaces, but METAFLOW_USER has a one to one mapping with KFP_SDK_USERID.